tools/hidl/hidlcomp.h

```diff
@@ -989,6 +997,34 @@ class EspMethodInfo
         response_ =strdup(name);
     }
 
+    bool isExecutionProfilingEnabled() const {return enableExecutionProfiling;}
+    void setEnableExecutionProfiling(bool enable)
+    {
+        enableExecutionProfiling = enable;
+
+        if (enable)
+        {
+            executionProfilingHistogramVariableName.append("p");
+            executionProfilingHistogramVariableName.append(name_);
+            executionProfilingHistogramVariableName.append("ScaledHistogram");
+        }
+
+    }
+
+    const std::string &getExecutionProfilingOptions(){return executionProfilingOptions;}
+    void setExecutionProfilingOptions(const char *options)
+    {
+        executionProfilingOptions = std::string(options);
```

Comment:
Curious why you can't simply?:

executionProfilingOptions = options;

Reply:
I generally am more explicit in areas like this so that when you are looking at the code you'd know that a std::string was being constructed. I changed it to be more concise.